Vagrantfile

```diff
@@ -13,6 +13,7 @@ $NETNEXT_SERVER_VERSION= (ENV['NETNEXT_SERVER_VERSION'] || $NETNEXT_SERVER_VERSI
 if ENV['NETNEXT'] == "true"
     $SERVER_BOX = $NETNEXT_SERVER_BOX
     $SERVER_VERSION = $NETNEXT_SERVER_VERSION
+    $vm_kernel = '+'
```

Comment:
Not sure if this is clear enough, but I wanted to keep the VM names as short as possible, since `vagrant` command-line completion is often buggy.